scripts/terminalserver/terminalserver.jl

```diff
@@ -16,26 +16,37 @@ module JSONRPC
 end
 
 include("misc.jl")
+include("trees.jl")
 include("repl.jl")
 include("../debugger/debugger.jl")
 
 const INLINE_RESULT_LENGTH = 100
 const MAX_RESULT_LENGTH = 10_000
 
-function getVariables()
+function get_variables()
     M = Main
     variables = []
+    clear_lazy()
+
     for n in names(M)
         !isdefined(M, n) && continue
         x = getfield(M, n)
-        x isa Module && continue
-        x==Main.vscodedisplay && continue
+        # x isa Module && continue
```

Comment:
same here, filter `Main` here to remove duplication.

Reply:
Done.